ipinfo/cmd_download.go

```diff
@@ -136,6 +146,26 @@ func cmdDownload() error {
 
 	url := fmt.Sprintf("%s%s.%s?token=%s", dbDownloadURL, dbName, format, token)
 	err := downloadDb(url, fileName, format, fZip)
+
+	// check if checksums match.
+	checksumUrl := fmt.Sprintf("%s%s.%s/checksums?token=%s", dbDownloadURL, dbName, format, token)
+	// fetch checksums from API.
+	checksumResponse, err := fetchChecksums(checksumUrl)
+	if err != nil {
+		return err
+	}
+
+	// compute checksum of downloaded file.
+	localChecksum, err := computeSHA256(fileName)
+	if err != nil {
+		return err
+	}
+
+	// compare checksums.
+	if localChecksum != checksumResponse.Checksums.SHA256 {
+		return errors.New("checksums do not match. File might be corrupted")
+	}
+
 	if err != nil {
```

Comment:
This error was supposed to be connected to the download step but now it's hanging out here